modules/kuberay-cluster/kuberay-autopilot-values.yaml

```diff
@@ -22,7 +22,7 @@
 image:
   # Replace this with your own image if needed.
   repository: rayproject/ray
-  tag: 2.6.1-py310-gpu
+  tag: 2.9.3-py310-gpu
```

Comment:
should we bump it for Standard as well to keep things consistent?

Reply:
That's going to be a larger, possibly breaking change, that affects multiple applications. I would rather do that in a separate PR.

Reply:
this needs a corresponding change to the notebook where we pip install ray ==2.7.x . We need to align the ray versions ?

Has this change been validate with the notebook e2e?

Reply:
+1 to what Saikat said, my bad for missing this in the review. the notebook uses the same ray version as the standard kuberay config, does this work for AP?